doc/releases/changelog-0.30.0.md

```diff
@@ -1,146 +1,285 @@
-  `grad_on_execution`.
+* An updated QNode return system has been introduced. PennyLane QNodes now return exactly what you 
+  tell them to! 🎉
   [(#3957)](https://github.com/PennyLaneAI/pennylane/pull/3957)
   [(#3969)](https://github.com/PennyLaneAI/pennylane/pull/3969)
+  [(#3946)](https://github.com/PennyLaneAI/pennylane/pull/3946)
+  [(#3913)](https://github.com/PennyLaneAI/pennylane/pull/3913)
+  [(#3914)](https://github.com/PennyLaneAI/pennylane/pull/3914)
+  [(#3934)](https://github.com/PennyLaneAI/pennylane/pull/3934)
 
-<h3>Improvements 🛠</h3>
+  This was an experimental feature introduced in version 0.25 of PennyLane that was enabled
+  via `qml.enable_return()`. Now, it's the default return system. Here's how it works:
 
-<h4>Next-generation device API</h4>
+  Consider the following circuit:
+
+  ```python
+  import pennylane as qml
+  
+  dev = qml.device("default.qubit", wires=1)
+  
+  @qml.qnode(dev)
+  def circuit(x):
+      qml.RX(x, wires=0)
+      return qml.expval(qml.PauliZ(0)), qml.probs(0)
+  ```
 
-* The `sample_state` function is added to `devices/qubit` that returns a series of samples based on a given
-  state vector and a number of shots.
-  [(#3720)](https://github.com/PennyLaneAI/pennylane/pull/3720)
+  In version 0.29 and earlier of PennyLane, `circuit()` would return a single length-3 array:
 
-* The `simulate` function added to `devices/qubit` now supports measuring expectation values of large observables such as
-  `qml.Hamiltonian`, `qml.SparseHamiltonian`, `qml.Sum`.
-  [(#3759)](https://github.com/PennyLaneAI/pennylane/pull/3759)
+  ```pycon
+  >>> circuit(0.5)
+  tensor([0.87758256, 0.93879128, 0.06120872], requires_grad=True)
+  ```
 
-* The `apply_operation` function added to `devices/qubit` now supports broadcasting.
-  [(#3852)](https://github.com/PennyLaneAI/pennylane/pull/3852)
+  In versions 0.30 and above, `circuit()` returns a length-2 tuple containing the expectation
+  value and probabilities separately:
 
-* `pennylane.devices.qubit.preprocess` now allows circuits with non-commuting observables.
-  [(#3857)](https://github.com/PennyLaneAI/pennylane/pull/3857)
+  ```pycon
+  >>> circuit(0.5)
+  (tensor(0.87758256, requires_grad=True),
+   tensor([0.93879128, 0.06120872], requires_grad=True))
+  ```
 
-* Adjoint differentiation support for the new qubit state-vector device has been added via
-  `adjoint_jacobian` in `devices/qubit`.
-  [(#3790)](https://github.com/PennyLaneAI/pennylane/pull/3790)
+  More details about this change, along with help and troubleshooting tips to solve any issues,
+  can be found [here](https://docs.pennylane.ai/en/stable/introduction/returns.html).
 
-* `qml.devices.qubit.measure` now computes the expectation values of `Hamiltonian` and `Sum`
-  in a backpropagation-compatible way.
-  [(#3862)](https://github.com/PennyLaneAI/pennylane/pull/3862/)
+<h4>A bunch of performance tweaks 🏃💨</h4>
```

Comment:
@trbromley I am splitting hairs here, but is this a better fit for the improvements section? I started reading this section, and every bullet point feels like an improvement 😆

Reply:
Yes probably! It was in the improvements section, but then I felt like there wasn't enough in the new features section so I brought it up. But if you are feeling this while reading through, it's a good datapoint to move back down.

Reply:
This has been fixed :+1: Some have been moved down to improvements, some have been expanded upon.